### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by Hellslicer**
- remove unneeded promises
- improve example projects

### CODE REVIEW

File: examples/javascript/README.md [hunk 1]
```diff
@@ -1,10 +1,4 @@
...
-This example compiles a JavaScript webpack project that you can use in your FiveM server. It allows a player to spawn an Adder vehicle at their current location and sets the player into the vehicle.
-
----
-
-### Usage
-
-`npm install` - Install required node modules.
-`npm run wc` - This will compile an `index.js` file into the `dist/` folder that you can use on your FiveM server.
+This example compiles a JavaScript webpack project that you can use in your FiveM server. It allows a player to spawn an Adder vehicle at their current location and sets the player into the vehicle.  
+Thanks to the [`yarn` and `webpack` resources](https://github.com/citizenfx/cfx-server-data/tree/master/resources/%5Bsystem%5D/%5Bbuilders%5D), dependencies will be installed and the project will be automagically compiled on resource start.
```
Comment (d0p3t) [inline examples/javascript/README.md:4]: `automagically` - spelling mistake

File: examples/typescript/README.md [hunk 2]
```diff
@@ -1,10 +1,4 @@
...
-This example compiles a Typescript webpack project that you can use in your FiveM server. It allows a player to spawn an Adder vehicle at their current location and sets the player into the vehicle.
-
----
-
-### Usage
-
-`npm install` - Install required node modules.
-`npm run wc` - This will compile an `index.js` file into the `dist/` folder that you can use on your FiveM server.
+This example compiles a Typescript webpack project that you can use in your FiveM server. It allows a player to spawn an Adder vehicle at their current location and sets the player into the vehicle.  
+Thanks to the [`yarn` and `webpack` resources](https://github.com/citizenfx/cfx-server-data/tree/master/resources/%5Bsystem%5D/%5Bbuilders%5D), dependencies will be installed and the project will be automagically compiled on resource start.
```
Comment (d0p3t) [inline examples/typescript/README.md:4]: same spelling mistake 😄

File: examples/javascript/package.json [hunk 3]
```diff
@@ -3,17 +3,10 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
-  "scripts": {
-    "wc": "webpack --config client.config.js"
```
Comment (d0p3t) [inline examples/javascript/package.json:7]: I'm not too sure whether I want to remove this setup.

I understand the ability for the script to automatically compile on a server, but this serves as an example on how to use `fivem-js`.

I recommend users to use the webpack-cli to build combined with a watcher and possibly quake3-rcon to automatically restart the resource to speed up development.

The built-in webpack/yarn builders are quite slow in my experience.

What's your opinion?

**Review by d0p3t — APPROVED** [review]
I'll merge for now including the updated examples. It will be more up to the standards of FiveM releases.

I will fix the spelling mistake myself

File: examples/javascript/package.json (hunk 3 above)
Comment (Hellslicer) [reply examples/javascript/package.json:7]: I wasn't aware about the slowness issue.
I agree, we can keep both if there is any use cases.

**Pull request merged**